src/Microsoft.SqlTools.ServiceLayer/QueryExecution/ResultSet.cs

```diff
@@ -613,45 +617,70 @@ protected virtual void Dispose(bool disposing)
         /// Sends the ResultsUpdated message if the number of rows has changed since last send.
         /// </summary>
         /// <param name="stateInfo"></param>
-        private async void SendResultAvailableOrUpdated (object stateInfo = null)
+        private void SendResultAvailableOrUpdated (object stateInfo = null)
```

Comment:
looks like this doesn't need the `stateInfo` parameter anymore

Reply:
This method is callback delegate signature of a TimerCallback method which requires this parameter in the signature.